.github/workflows/lint.yaml

```diff
@@ -33,5 +33,8 @@ jobs:
           uds run lint:deps --no-progress
 
       - name: Lint the repository
+        env:
+          OSCALFILES: ./oscal.yaml
```

Comment:
```suggestion
```
(if the default variable suggestion is taken below)

Reply:
removed